packages/cli/src/services/orchestration/main/handleCommandMessageMain.ts

```diff
@@ -77,6 +78,20 @@ export async function handleCommandMessageMain(messageString: string) {
 				}
 				await Container.get(ExternalSecretsManager).reloadAllProviders();
 				break;
+			case 'community-package-install':
+			case 'community-package-update':
+			case 'community-package-uninstall':
+				if (!debounceMessageReceiver(message, 200)) {
+					return message;
+				}
+				const { packageName, packageVersion } = message.payload;
+				const communityPackagesService = Container.get(CommunityPackagesService);
+				if (message.command === 'community-package-uninstall') {
+					await communityPackagesService.removeNpmPackage(packageName);
+				} else {
+					await communityPackagesService.installOrUpdateNpmPackage(packageName, packageVersion);
+				}
+				break;
```

Comment:
So this should work for multi-main as well right?

Reply:
I believe so. but I haven't tested this bit.